components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -387,6 +458,21 @@ public RuleTemplate getRuleTemplate(String templateGroupUUID, String ruleTemplat
         return templates;
     }
 
+    public Collection<Template> getTemplates(BusinessRuleFromScratch businessRuleFromScratch) throws
+            TemplateManagerException {
+
+        List<RuleTemplate> foundRuleTemplate = getRuleTemplate(businessRuleFromScratch);
+        //Collection<Template> templates=null;
```

Comment:
Remove commented out code

Reply:
This was fixed by b53d410a315b24891f480ea4b7e851caddb2e6cf